client/app/telecom/pack/xdsl/modem/pack-xdsl-modem.html

```diff
@@ -129,6 +128,18 @@ <h2 class="widget-presentation-title" data-translate="xdsl_modem_advanced_config
                         </div>
                     </div>
                 </div>
+
+                <div class="my-3" data-ng-class="{ 'disabled' : !XdslModem.mediator.capabilities.canChangeFirmware || XdslModem.mediator.tasks.changeModemConfigFirmware }">
+                    <label for="configFirmware">
+                        <span class="tooltip-label"
```

Comment:
Does `<oui-tooltip>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip) could fit your needs?

I know that it was usually done in that way but If we can replace it with our component that would be great.